File: syntax-simplicity/syntax_simplicity.py
def is_even(n):
    return n % 2 == 0


def main():
    numbers = [5, 1, 4, 2, 3]
    even_squares = []

    for n in numbers:
        if is_even(n):
            even_squares.append(n * n)

    total = sum(even_squares)

    print("Language: Python")
    print(f"Input: {numbers}")
    print(f"Even squares: {even_squares}")
    print(f"Total: {total}")


if __name__ == "__main__":
    main()

File: syntax-simplicity/SyntaxSimplicity.cs
using System;
using System.Collections.Generic;

class SyntaxSimplicity
{
    static bool IsEven(int n)
    {
        return n % 2 == 0;
    }

    static void Main()
    {
        var numbers = new List<int> { 5, 1, 4, 2, 3 };
        var evenSquares = new List<int>();

        foreach (var n in numbers)
        {
            if (IsEven(n))
            {
                evenSquares.Add(n * n);
            }
        }

        var total = 0;
        foreach (var value in evenSquares)
        {
            total += value;
        }

        Console.WriteLine("Language: C#");
        Console.WriteLine($"Input: [{string.Join(", ", numbers)}]");
        Console.WriteLine($"Even squares: [{string.Join(", ", evenSquares)}]");
        Console.WriteLine($"Total: {total}");
    }
}
